compliance_checker/tests/test_cli.py

```diff
@@ -20,6 +20,13 @@
 
 from .conftest import datadir, static_files
 
+on_windows = platform.system() == "Windows"
+
+if on_windows:
+    ncconfig = ["sh", f"{os.environ['CONDA_PREFIX']}\\Library\\bin\\nc-config"]
+else:
+    ncconfig = ["nc-config"]
```

Comment:
@benjwadams while this works, I'm not sure we need this. We can find the version with netCDF4 and let the zarr  availability be checked by the failing test. What do you think?